frontend/packages/kubevirt-plugin/src/components/modals/vm-boot-order-modal/components/dnd-list/dnd-list.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { DataList } from '@patternfly/react-core';
+import { PficonDragdropIcon } from '@patternfly/react-icons';
+import { DndProvider } from 'react-dnd';
+import HTML5Backend from 'react-dnd-html5-backend';
+import { DNDRowAddItem } from './dnd-row-add-item';
+import { DNDRow } from './dnd-row';
+
+import './dnd-list.scss';
+
+export const DNDList: React.FC<DNDListProps> = ({
+  Component,
+  size,
+  onDelete,
+  onMove,
+  canAddItem,
+  AddItemComponent,
+  addItemMessage,
+  addItemIsDisabled,
+  AddItemIsEditMode,
+  addItemDisabledMessage,
+  addItemSetEditMode,
+}) => (
+  <DndProvider backend={HTML5Backend}>
+    <DataList aria-label="sorted list">
+      {[...Array(size).keys()].map((index) => (
```

Comment:
you can use `_.range` instead

Reply:
:tada:  thanks :+1:, looked for something like that, and missed it  :-)